docker/mediamtx-entry.bash

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+[ -v DEBUG ] && set -x
+
+set -euo pipefail
+
+if [ -v LP_PUBLISH_MEDIAMTX_METRICS ]; then
+  if [ -z "$LP_PUBLISH_MEDIAMTX_METRICS_ENDPOINT" ]; then
```

Comment:
Should we exit if the endpoint is not specified?
```bash
if [ -z "$LP_PUBLISH_MEDIAMTX_METRICS_ENDPOINT" ]; then
    echo >&2 "No endpoint specified for publishing mediamtx metrics."
    exit 1
fi
```

Reply:
i was thinking it more as a warning than a fail condition.

Reply:
if `LP_PUBLISH_MEDIAMTX_METRICS_ENDPOINT` is missing then `curl` would fail few lines below.

But yeah warning is fine... I think we'd catch it missing anyway as not having mediamtx version on grafana